src/Event/ApiGateway/WebsocketEvent.php

```diff
@@ -0,0 +1,141 @@
+<?php declare(strict_types=1);
+
+namespace Bref\Event\ApiGateway;
+
+use Bref\Event\InvalidLambdaEvent;
+use Bref\Event\LambdaEvent;
+
+/**
+ * Represents a Lambda event when Lambda is invoked by ApiGateway websocket route.
+ */
+final class WebsocketEvent implements LambdaEvent
+{
+    /** @var array */
+    private $event;
+
+    /** @var string */
+    private $routeKey;
+
+    /** @var int */
+    private $eventType;
+
+    /** @var mixed */
+    private $body;
```

Comment:
```suggestion
    /** @var mixed|null */
    private $body = null;
```

That was the property is always initialized (thinking of PHP 7.4+ migration), and this will simplify the getter below (see my next comment)

Reply:
I agree. Much better approach.